app/src/main/java/org/systers/mentorship/view/activities/MainActivity.kt

```diff
@@ -72,6 +74,13 @@ class MainActivity : BaseActivity() {
                 R.id.menu_feedback -> {
                     true
                 }
+                R.id.menu_logout -> {
+                    PreferenceManager().clearPreferences()
```

Comment:
create an instance of `PreferenceManager`at the top of the file and use it here.

Just like it has been done [here](https://github.com/systers/mentorship-android/blob/develop/app/src/main/java/org/systers/mentorship/viewmodels/LoginViewModel.kt#L45)